.gitignore

```diff
@@ -153,3 +153,147 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
+# Created by https://www.toptal.com/developers/gitignore/api/node
+# Edit at https://www.toptal.com/developers/gitignore?templates=node
```

Comment:
I somewhat favour keeping gitignore as local as posisble, so I would rather have it inside images/techdocs-cli/context, but nothing wrong with it here either.

Reply:
fixed